lib/features/children/generosity_challenge/pages/generosity_challenge_day_details.dart

```diff
@@ -70,10 +70,11 @@ class GenerosityChallengeDayDetails extends StatelessWidget {
                     ),
                   ),
                 ),
-              if (!isDualCard ||
-                  state.status ==
-                      GenerosityChallengeStatus.dailyAssigmentConfirm ||
-                  day.isCompleted)
+              if (challenge.state.islastDay &
+                  (!isDualCard ||
```

Comment:
For context, what does "isDualCard" mean, when does this occur in the app/challenge?

Reply:
It occurs on day 2 and day 7, it is when the assignment screen does not have a complete button. They see an assignment the follow a flow (select values/ give to org) then they come back to the assignment screen, see a dynamic descriptio and can click on the complete button. But i will also leave a comment in the code, try to make that logic clearer

Reply:
Clear, thanks!